app/src/main/java/dao/IRetrofitWeatherDAO.kt

```diff
@@ -1,11 +1,12 @@
 package dao
 
 import dto.CurrentWeather
+import dto.Forecast
 import retrofit2.Call
 import retrofit2.http.GET
 
 interface IRetrofitWeatherDAO {
     @GET("/v1/geocode/{latitude}/{longitude}/forecast/hourly/{12}hour.json")
     fun getGeoCode(): Call<ArrayList<CurrentWeather>>
-    fun getAllForecast(): Any
+    fun getAllForecast(): ArrayList<Forecast>
```

Comment:
There were issues further on when this was set to any, I felt it would make more sense to be an array of forecasts

Reply:
That type is a bit more precise.  You can probably use the interface List instead of the class ArrayList, as we tend to use interfaces as variable types, for polymorphism.